config/profile.go

```diff
@@ -25,3 +36,80 @@ func (c *Config) RemoveProfile(name string) {
 
 	delete(c.Profiles, name)
 }
+
+func (c *Config) ParseProfileInput(input string) (conn, key string, err error) {
+	c.Lock()
+	defer c.Unlock()
+
+	sepIdx := strings.LastIndex(input, profileSeparator)
+
+	// separator not found, treat as normal fs path
+	// 1. if path is absolute path, set work dir is /
+	// 2. if path is relative path, set work dir is current dir
+	// in both condition, treat path as object key
+	if sepIdx == -1 {
+		prefix := "fs://"
+		// absolute path
+		if strings.HasPrefix(input, "/") {
+			conn = prefix + "/"
+		} else {
+			curDir, err := os.Getwd()
+			if err != nil {
+				return "", "", err
+			}
+			conn = prefix + curDir
+		}
+
+		key = input
+		return
+	}
+
+	// handle profile
+	name := input[:sepIdx]
+	prof, ok := c.Profiles[name]
+	// profile not exist, treat as full connection string
+	if !ok {
+		return "", "", fmt.Errorf("profile with name %s not exist", name)
+	}
+
+	conn = prof.Connection
+
+	// handle key
+	// if input end with ':', set key to blank
+	if sepIdx == len(input)-1 {
+		key = ""
+	} else {
+		key = input[sepIdx+1:]
+	}
+	return
+}
+
+func (c *Config) MergeProfileFromEnv() {
+	c.Lock()
+	defer c.Unlock()
+
+	env := getEnvWith(profileEnvPrefix)
+	for k, v := range env {
+		c.Profiles[k] = Profile{Connection: v}
+	}
+}
+
+// getEnvWith get env var with given prefix and return a map with key trimmed prefix
+func getEnvWith(prefix string) map[string]string {
```

Comment:
If we only use this function once, we can merge into `MergeProfileFromEnv` directly.